src/app/pages/admin/twins/twins.component.ts

```diff
@@ -0,0 +1,135 @@
+import { Component, OnInit } from '@angular/core';
+import { Observable } from 'rxjs';
+
+import { LocalDataSource } from 'ng2-smart-table';
+
+import { NbDialogService } from '@nebular/theme';
+import { ConfirmationComponent } from 'app/shared/confirmation/confirmation.component';
+import { DetailsComponent } from 'app/shared/details/details.component';
+
+import { TwinsService } from 'app/common/services/twins/twins.service';
+import { Twin } from 'app/common/interfaces/mainflux.interface';
+
+
+@Component({
+  selector: 'ngx-twins',
+  templateUrl: './twins.component.html',
+  styleUrls: ['./twins.component.scss'],
+})
+export class TwinsComponent implements OnInit {
+  settings = {
+    add: {
+      addButtonContent: '<i class="nb-plus"></i>',
+      createButtonContent: '<i class="nb-checkmark"></i>',
+      cancelButtonContent: '<i class="nb-close"></i>',
+      confirmCreate: true,
+    },
+    edit: {
+      editButtonContent: '<i class="nb-edit"></i>',
+      saveButtonContent: '<i class="nb-checkmark"></i>',
+      cancelButtonContent: '<i class="nb-close"></i>',
+      confirmSave: true,
+    },
+    delete: {
+      deleteButtonContent: '<i class="nb-trash"></i>',
+      confirmDelete: true,
+    },
+    columns: {
+      id: {
+        title: 'ID',
+        editable: false,
+        addable: false,
+        filter: false,
+      },
+      thing_id: {
+        title: 'Thing ID',
```

Comment:
Thing or Twin here?

Reply:
Thing ID. It's a thing connected to a twin.